atd-toolbox/load_council_districts/data/README.md

```diff
@@ -0,0 +1,5 @@
+This geoJSON file is an intermediate product from loading the council districts into
+the Vision Zero database. The upstream, and up to date, source of this data is found here:
+https://austin.maps.arcgis.com/home/item.html?id=8251869f87e24d3abff3d7a36fc7c4f4.
```

Comment:
weird behavior where if i ctrl + c this link here i get a 403 forbidden error but if i copy and paste it, it works. can anyone explain this bizarreness for my own curiosity?

Reply:
weird! just a guess, but it might maybe AGOL doesn't like the `Referer` header when click from here?